bittensor/_subtensor/subtensor_impl.py

```diff
@@ -865,10 +867,9 @@ def add_stake_multiple (
                 # Stake it all.
                 staking_balance = bittensor.Balance.from_tao( old_balance.tao )
                 staking_all = True
-
-            elif not isinstance(amount, bittensor.Balance ):
-                staking_balance = bittensor.Balance.from_tao( amount )
```

Comment:
don't we need this anymore?

Reply:
We cast above to Balance so they are all Balances